packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1009,6 +1009,8 @@ abstract class BoxScrollView extends ScrollView {
 /// `ListView` has no built-in notion of a selected item or items. For a small
 /// example of how a caller might wire up basic item selection, see
 /// [ListTile.selected].
+/// For a custom implementation of ListTile selection in a ListView, please refer the example at
+/// [examples/api/lib/widgets/scroll_view/listview_select.1.dart](https://github.com/flutter/flutter/tree/master/examples/api/lib/widgets/scroll_view/listview_select.1.dart)
```

Comment:
This should be included in a {@tool dartpad... like the other example. :)
That is what makes the sample in the api docs interactive.

Reply:
Sure thing ! 
Done.